spk/rutorrent/src/rtorrent.rc

```diff
@@ -13,3 +13,5 @@ dht.mode.set = auto
 dht.port.set = 6881
 protocol.pex.set= yes
 port_range = 6881-6999
+schedule2 = session_save, 1200, 3600, ((session.save))
+method.set_key = event.download.inserted, 2_save_session, ((d.save_full_session))
```

Comment:
Open question: this will only be applied on new installation and won't be applied on upgrade. Is this something we want or would we want to try to amend existing installation's configurations on upgrade ?

Reply:
Yes, session saving could present challenges for NAS environments. It highly possible to consume more resources writing session files with torrent information to disk than actually seeding torrents. RAID 5 in particular is very suspectable to this issue.

This and the potential to lose torrents is why I would recommend amending existing installs.